templates/types/streaming/express/index.ts

```diff
@@ -31,8 +31,8 @@ if (isDevelopment) {
   console.warn("Production CORS origin not set, defaulting to no CORS.");
 }
 
-app.use("/api/data", express.static("data"));
-app.use("/api/tool-output", express.static("tool-output"));
+app.use("/api/files/data", express.static("data"));
```

Comment:
where are the changes in the source node viewer that we changed /api/data to api/files/data?